src_c/_pygame.h

```diff
@@ -108,6 +108,33 @@ PG_UnlockMutex(SDL_mutex *mutex)
 #define PG_FORMAT_BytesPerPixel(format) format->bytes_per_pixel
 #define PG_SURF_FORMATENUM(surf) surf->format
 
+#define PG_FORMAT_R_LOSS(format) format->Rbits
+#define PG_FORMAT_G_LOSS(format) format->Gbits
+#define PG_FORMAT_B_LOSS(format) format->Bbits
+#define PG_FORMAT_A_LOSS(format) format->Abits
```

Comment:
```suggestion
#define PG_FORMAT_R_LOSS(format) (8 - format->Rbits)
#define PG_FORMAT_G_LOSS(format) (8 - format->Gbits)
#define PG_FORMAT_B_LOSS(format) (8 - format->Bbits)
#define PG_FORMAT_A_LOSS(format) (8 - format->Abits)
```
bits and loss are not the same thing, but they are complementary. Ref: https://github.com/libsdl-org/SDL/pull/10187

Reply:
Good spot.